doc/dev_guide/testing.rst

```diff
@@ -109,6 +109,17 @@ that all tests in a file run in the same worker.
     `pytest-sugar <https://pypi.org/project/pytest-sugar/>`_ to produce
     nicer-looking output including an animated progressbar.
 
+To test docstring examples, assuming the current location is the HyperSpy root
+directory:
+
+.. code:: bash
+
+    # All
+    $ pytest --doctest-modules --ignore-glob=hyperspy/tests hyperspy/*.py
```

Comment:
This pattern doesn't seem to work on windows.

```suggestion
    $ pytest --doctest-modules --ignore-glob=hyperspy/tests hyperspy
```

An alternative could be to use `--pyargs`:
```suggestion
    $ pytest --doctest-modules --ignore-glob=hyperspy/tests --pyargs hyperspy
```

Reply:
Thank you for pointing it out. I've removed the wildcard and added `pyargs` as you suggested. This gives the same behaviour on Linux as my original command.